react/hello-world/src/components/pages/home/index.jsx

```diff
@@ -0,0 +1,28 @@
+import './Home.scss'
+import { useState } from 'react'
+import { FaPlus, FaMinus } from 'react-icons/fa'
+import Button from '../../button'
+
+const Home = () => {
+  const [counter, setCounter] = useState(0)
```

Comment:
define type

Reply:
did you mean that I should write it in TypeScript?

Reply:
yes, you learn TS before, so please apply it